.doc_gen/metadata/iam_metadata.yaml

```diff
@@ -2186,6 +2389,21 @@ iam_Scenario_CreateUserAssumeRole:
     - Delete the policy, role, and user.
   category: Scenarios
   languages:
+    .NET:
+      versions:
+        - sdk_version: 3
+          github: dotnetv3/IAM/IAM_Basics_Scenario
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - IAM.dotnetv3.IAM_BasicsScenario
+                - IAM.dotnetv3.CreateUserAsync
+                - IAM.dotnetv3.CreateAccessKey
+                - IAM.dotnetv3.CreatePolicyAsync
+                - IAM.dotnetv3.AttachPolicy
+                - IAM.dotnetv3.CreateRoleAsync
+                - IAM.dotnetv3.DeleteResourcesAsync
```

Comment:
Remove all but the first of these. All of these snippets are already included in the main IAM_BasicsScenario snippet, so including them a second time has repeated them in the output.

Reply:
Done